builtin/fsmonitor--daemon.c

```diff
@@ -631,7 +631,7 @@ static int fsmonitor_parse_client_token(const char *buf_token,
 	return 0;
 }
 
-KHASH_INIT(str, const char *, int, 0, kh_str_hash_func, kh_str_hash_equal);
+KHASH_INIT(str, const char *, int, 0, kh_str_hash_func, kh_str_hash_equal)
```

Comment:
This has been squashed into builtin-fsmonitor-part3 that I'm waiting to send to the list as soon as part 2 gets moved into seen.

Reply:
At that time, we might want to consider integrating a new iteration of the FSMonitor series into Git for Windows...

Reply:
That would be good.  Hey, wait, you're on vacation. Go away.

Reply:
> 
> 
> That would be good. Hey, wait, you're on vacation. Go away.

I did! Going away was the whole point of the vacation ;-)

Also: thanks for opening #3447 (which I merged)!